benchmarking/streamlit/app.py

```diff
@@ -38,75 +39,31 @@ def _run_performance_evaluation() -> pd.DataFrame:
     """
 
     results_path = "./data/results/llmperf"
-    num_concurrent_workers = st.session_state.number_concurrent_workers
 
-    # Call benchmarking process. Static param values are intentional and still WIP.
-    mode = "stream"
-    llm_api = "sambastudio"
-
-    run_token_benchmark(
-        model=st.session_state.llm,
-        num_input_tokens=st.session_state.input_tokens,
-        num_output_tokens=st.session_state.output_tokens,
-        timeout_s=st.session_state.timeout,
-        max_num_completed_requests=st.session_state.number_requests,
-        num_concurrent_workers=num_concurrent_workers,
-        additional_sampling_params="{}",
+    # Call benchmarking process
+    performance_evaluator = SyntheticPerformanceEvaluator(
```

Comment:
Instantiate the relevant sub-class for performance eval with a synthetic dataset generated based on input params from user.